app/src/main/java/io/github/droidkaigi/confsched2018/util/ProgressTimeLatch.kt

```diff
@@ -0,0 +1,52 @@
+package io.github.droidkaigi.confsched2018.util
```

Comment:
Please add google license 🏷 
https://github.com/chrisbanes/tivi/blob/96e7cae7560ffd358b8c58c47267ed1024df53f6/app/src/main/java/me/banes/chris/tivi/ui/ProgressTimeLatch.kt
And can you add to README?